django_project/base/models/project.py

```diff
@@ -4,6 +4,7 @@
 import logging
 import string
 import re
+import unidecode
```

Comment:
You don't need to import this one if you will only use unidecode and import it also below